recipe/meta.yaml

```diff
@@ -10,16 +10,16 @@ source:
     sha256: 36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d
 
 build:
-    number: 1
+    number: 3
```

Comment:
Build numbers should become irrelevant when comparing versions across channels: https://github.com/conda/conda/issues/2779#issuecomment-228626696